app/survey_config/business_config.py

```diff
@@ -17,6 +17,14 @@ class BusinessSurveyConfig(
     footer_legal_links: Iterable[Mapping] = field(default_factory=list)
 
     def __post_init__(self):
+        if self.schema:
+            self.data_layer: Iterable[Mapping] = [
```

Comment:
Anything we want to do with a property that is inherited should be after the `post_init` call, i.e post init might override or do something with it. Prior only if we want to preprocess before calling post init, such as the base url.

Reply:
Moved after `post_init`, not always 100% how it works, thanks.